SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -307,6 +315,9 @@ rm -f %{pkidir}/tls/certs/*.{0,pem}
 %dir %{pkidir}/tls
 %dir %{pkidir}/tls/certs
 
+# Distrusted CAs
+%{_datadir}/pki/ca-trust-source/%{p11_format_distrusted_bundle}
```

Comment:
Do we need to do anything special here regarding file permissions.  We definitely would not want the "distrusted list" to be deleted by some rogue actors

Reply:
Good question. The whole `%{_datadir}/pki` directory and its children are read-only for non-root users. Having said that, this is probably mostly due to the fact, that we build the RPMs as root. Let me see, if I can make this more explicit.

Reply:
I've now explicitly set the file ownership to `root:root`, so regular users won't be able to modify these files even if we change how we build our RPMs.